tests/conftest.py

```diff
@@ -170,6 +193,23 @@ def get_fixtures():
     return cloudbeat_agent, k8s
 
 
+def pytest_sessionstart():
+    """
+    Called after the Session object has been created and before performing collection and entering the run test loop.
+    @return:
+    """
+
```

Comment:
PEP 257: [There's no blank line either before or after the docstring.](https://peps.python.org/pep-0257/) not sure why our linter didn't catch that

Reply:
removed